lighthouse-core/audits/byte-efficiency/uses-optimized-images.js

```diff
@@ -96,9 +96,11 @@ class UsesOptimizedImages extends ByteEfficiencyAudit {
           continue;
         }
 
+        // Skip if we couldn't collect natural image size information.
+        if (!imageElement.naturalDimensions) continue;
```

Comment:
ha, adapting existing code is kind of awkward, but this is really nice to have an explicit guard if there wasn't natural image size info and not have to worry about falsy values, etc